frontend/redux/sagas.js

```diff
@@ -36,11 +38,22 @@ export function* requestWorkload() {
   }
 }
 
+export function* completeWorkload() {
+  try {
+    const workloadToSend = yield select(selectWorkload);
+    const workloadPostResponse = yield call(postWorkload, workloadToSend);
+    yield put(workloadPostResponse);
+  } catch (e) {
+    console.log('I FAILED TO POST ANNOTATIONS', e);
```

Comment:
We should probably start sending some action on the catch block of these sagas --- cc @kadamwhite something like `yield put( postWorkloadFailed(e, workloadToSend) )` or something (`postWorkloadFailed` being an action creator?)

This should not block this PR from landing